prov/efa/src/rxr/rxr_pkt_type_misc.c

```diff
@@ -490,11 +490,23 @@ void rxr_pkt_handle_eor_recv(struct rxr_ep *ep,
 {
 	struct rxr_eor_hdr *shm_eor;
 	struct rxr_tx_entry *tx_entry;
+	int i, ret;
 
 	shm_eor = (struct rxr_eor_hdr *)pkt_entry->pkt;
 
 	/* pre-post buf used here, so can NOT track back to tx_entry with x_entry */
 	tx_entry = ofi_bufpool_get_ibuf(ep->tx_entry_pool, shm_eor->tx_id);
+	if (!efa_cma_cap) {
+		for (i = 0; i < tx_entry->rma_iov_count; i++) {
+			if (tx_entry->mr[i]) {
+				ret = fi_close(&tx_entry->mr[i]->fid);
+				if (ret) {
```

Comment:
OFI_UNLIKELY

Reply:
Fixed

Reply:
These has been done in https://github.com/ofiwg/libfabric/pull/5721 the 3rd commit. We need to compare notes on this

Reply:
Updated with the one in #5721.